lua/lspconfig/server_configurations/texlab.lua

```diff
@@ -19,6 +19,7 @@ local function buf_build(bufnr)
   local texlab_client = util.get_active_client_by_name(bufnr, 'texlab')
   local params = {
     textDocument = { uri = vim.uri_from_bufnr(bufnr) },
+    position = { line = vim.fn.line '.' - 1, character = vim.fn.col '.' },
```

Comment:
```lua
local pos = vim.api.nvim_win_get_cursor(0)
pos[1] -1 is line pos[2] is col
```

Reply:
Should I also change the other times it appears? It's like that in all the file.

Reply:
@glepnir changed :)